src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -82,7 +82,7 @@ public static Index parseIndex(String oneBasedIndex) throws IllegalValueExceptio
      * See header comment of this class regarding the use of {@code Optional} parameters.
      */
     public static Optional<Phone> parsePhone(Optional<String> phone) throws IllegalValueException {
-        requireNonNull(phone);
+        //requireNonNull(phone);
         return phone.isPresent() ? Optional.of(new Phone(phone.get())) : Optional.empty();
```

Comment:
Can just return that Optional straight away, rather than returning a new instance of an Optional with the same value.
Likewise for the other optional fields